### PR TITLE
added cartPage, cartItem, useCart hook and changed some styles

### PR DESCRIPTION
**Pull request by Vatrushka20**

### CODE REVIEW

**Review by KonstantinLigus — CHANGES_REQUESTED** [review]

File: src/frontend/components/consumers/CartItem/CartItem.jsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        </Link>
+        <p className={styles.price}>{price} грн</p>
+        <p className={styles.measure}>за 100 грам</p>
+
+        <div className={styles.counter}>
+          <div className={styles.amount}>
+            <div className={styles.amountIconContainer}>
+              <Image
+                className={styles.amountIcon}
+                src={minusIcon}
```
[the remark this answers is not in the document]
Comment (Vatrushka20) [reply src/frontend/components/consumers/CartItem/CartItem.jsx:43]: fixed

File: src/frontend/components/consumers/CartItem/CartItem.jsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import Link from "next/link";
+import Image from "next/image";
+import styles from "./CartItem.module.scss";
+import minusIcon from "../../../../../public/assets/icon/icons-minus.png";
+import plusIcon from "../../../../../public/assets/icon/icons-plus.svg";
```
[the remark this answers is not in the document]
Comment (Vatrushka20) [reply src/frontend/components/consumers/CartItem/CartItem.jsx:6]: fixed

File: src/frontend/components/consumers/CartItem/CartItem.jsx
[the remark this answers is not in the document]
Comment (Vatrushka20) [reply src/frontend/components/consumers/CartItem/CartItem.jsx:1]: agreed on the name in personal messages

**Review by Kulyk-Volodymyr — APPROVED** [review]

File: src/frontend/components/consumers/CartItem/CartItem.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (KonstantinLigus) [reply src/frontend/components/consumers/CartItem/CartItem.jsx:43]: still occured, ou should add some breackpoint to `size` prop

**Review by KonstantinLigus — APPROVED** [review]

**Pull request merged**